### DISCUSSION TITLE
Here is calendar issue too

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17207] [How to]
Added calendar to related list of custom module and this relation returning error. Log shows as per below. How to overcome this issue?

SQLSTATE[HY000]: General error: 3065 Expression #2 of ORDER BY clause is not in SELECT list, references column 'reeidcrm.vtiger_activity.time_start' which is not in SELECT list; this is incompatible with DISTINCT

**Comment by rskrzypczak** [comment]
You can try removing ONLY_FULL_GROUP_BY from sql_mode (see: https://bugs.mysql.com/bug.php?id=106813)
What DB version do you have?

**Reply by garryatbrighton** [reply]
Removed ONLY_FULL_GROUP_BY :Doesn't works
mysql  Ver 8.0.33
